src/app/+item-page/edit-item-page/edit-item-page.routing.module.ts

```diff
@@ -10,13 +10,23 @@ import { ItemDeleteComponent } from './item-delete/item-delete.component';
 import { ItemStatusComponent } from './item-status/item-status.component';
 import { ItemMetadataComponent } from './item-metadata/item-metadata.component';
 import { ItemBitstreamsComponent } from './item-bitstreams/item-bitstreams.component';
+import { ItemMoveComponent } from './item-move/item-move.component';
+import { URLCombiner } from '../../core/url-combiner/url-combiner';
+import { getItemEditPath } from '../item-page-routing.module';
 
 const ITEM_EDIT_WITHDRAW_PATH = 'withdraw';
 const ITEM_EDIT_REINSTATE_PATH = 'reinstate';
 const ITEM_EDIT_PRIVATE_PATH = 'private';
 const ITEM_EDIT_PUBLIC_PATH = 'public';
 const ITEM_EDIT_DELETE_PATH = 'delete';
 
+const ITEM_EDIT_MOVE_PATH = 'move';
+
+export function getItemEditMovePath(id: string) {
```

Comment:
Why does ITEM_EDIT_MOVE_PATH require a separate function here?  This function doesn't seem to be used anywhere.

Reply:
Removed this method since it was not used.